diff --git a/node_modules/@opennextjs/cloudflare/dist/cli/build/patches/plugins/load-manifest.js b/node_modules/@opennextjs/cloudflare/dist/cli/build/patches/plugins/load-manifest.js
--- a/node_modules/@opennextjs/cloudflare/dist/cli/build/patches/plugins/load-manifest.js
+++ b/node_modules/@opennextjs/cloudflare/dist/cli/build/patches/plugins/load-manifest.js
@@ -31,7 +31,7 @@
     const { outputDir } = buildOpts;
     const baseDir = join(outputDir, "server-functions/default", getPackagePath(buildOpts));
     const dotNextDir = join(baseDir, ".next");
-    const manifests = await glob(join(dotNextDir, "**/{*-manifest,required-server-files,prefetch-hints}.json"), {
+    const manifests = await glob(join(dotNextDir, "**/{*-manifest,required-server-files,prefetch-hints,preview-props}.json"), {
         windowsPathsNoEscape: true,
     });
     const returnManifests = (await Promise.all(manifests.map(async (manifest) => `
